extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -510,22 +525,33 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 				CSSStyles: { ...styles.BODY_CSS }
 			}).component();
 
-		const instructions = utils.createRegistrationInstructions(this._view, false);
-
-		this._authKeyTable = utils.createAuthenticationKeyTable(this._view, '50px', '500px');
+		const refreshAndConfigureIRcontainer = this._view.modelBuilder.toolbarContainer()
+			.withToolbarItems(
+				[
+					{ component: this._refreshButton },
+					{ component: this._configureIRButton }
+				]
+			).component();
 
-		statusContainer.addItems([
-			this._dmsStatusInfoBox,
-			instructions,
-			this._authKeyTable]);
+		this._integrationRuntimeTable = utils.createIntegrationRuntimeTable(this._view);
 
 		container.addItems([
 			connectionLabelContainer,
-			statusContainer]);
+			this._dmsStatusInfoBox,
+			refreshAndConfigureIRcontainer,
+			this._integrationRuntimeTable
+			//statusContainer
+		]);
 
 		return container;
 	}
 
+	// opens IR dialog
+	public async openIRDialog(): Promise<void> {
+		const configureIR = new ConfigureIRDialog(this.migrationStateModel);
+		await configureIR.openDialog();
```

Comment:
You may add a try catch here to handle and log error if IR dialog opening fails due to any error.